role-make-version-changelog.sh

```diff
@@ -209,7 +213,7 @@ if [ "$skip" = false ]; then
             ver_minor=$((ver_minor+=1))
             ver_patch=0
         fi
-    elif grep -q '^[a-z0-9][a-z0-9]* feat.*' $rev_commits_file; then
+    elif grep -q '^[a-z0-9][a-z0-9]* feat.*' $pr_titles_file; then
```

Comment:
I guess this will be different - the format won't be `abcdef1 feat: something` since you don't have a commit hash anymore